CONTRIBUTING.md

```diff
@@ -18,9 +18,9 @@ If you like to contribute code, squash a bug but if you don't know where to star
 
     This is a place to find feature requests, bugs.
 
-    Issues with the ```good first issue``` tag are good place for beginners to take on.
+    Issues with the `good first issue` tag are good place for beginners to take on.
```

Comment:
I've corrected these to single ` as it was messing with the Vim syntax coloring.